model/flow/ledger.go

```diff
@@ -14,22 +14,11 @@ type RegisterID struct {
 	Key   string
 }
 
-// this function returns a string format of a RegisterID in the form '%x/%x'
-// it has been optimized to avoid the memory allocations inside Sprintf
+// String returns formatted string representation of the RegisterID.
+// TODO(rbtz): remove after the merge of https://github.com/onflow/flow-emulator/pull/235
+// Deprecated: used only by emultor.
 func (r *RegisterID) String() string {
-	ownerLen := len(r.Owner)
-
-	requiredLen := ((ownerLen + len(r.Key)) * 2) + 1
-
-	arr := make([]byte, requiredLen)
-
-	hex.Encode(arr, []byte(r.Owner))
-
-	arr[2*ownerLen] = byte('/')
-
-	hex.Encode(arr[(2*ownerLen)+1:], []byte(r.Key))
-
-	return string(arr)
+	return fmt.Sprintf("%x/%x", r.Owner, r.Key)
```

Comment:
I'm not sure why we were doing it this way in the first place, I think it suppose to be some sort of optimized way of doing it

Reply:
yeah. It is about 2x fast as the `Sprintf`.  The main problem here is that it was nod needed at all -- we were using this for map indexing, instead of just using the struct itself (which is infinitely faster and noes not require allocations.)

Reply:
got it, then all good.